realm/src/main/java/io/realm/RealmObject.java

```diff
@@ -16,53 +16,60 @@
 
 package io.realm;
 
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.Future;
+
 import io.realm.annotations.RealmClass;
-import io.realm.internal.Row;
 import io.realm.internal.InvalidRow;
+import io.realm.internal.Row;
+import io.realm.internal.Table;
+import io.realm.internal.TableQuery;
+import io.realm.internal.log.RealmLog;
 
 /**
  * In Realm you define your model classes by sub-classing RealmObject and adding fields to be
  * persisted. You then create your objects within a Realm, and use your custom subclasses instead
  * of using the RealmObject class directly.
- * <p>
+ * <p/>
  * An annotation processor will create a proxy class for your RealmObject subclass. The getters and
  * setters should not contain any custom code of logic as they are overridden as part of the annotation
  * process.
- * <p>
+ * <p/>
  * A RealmObject is currently limited to the following:
- *
+ * <p/>
  * <ul>
- *   <li>Private fields.</li>
- *   <li>Getter and setters for these fields.</li>
- *   <li>Static methods.</li>
+ * <li>Private fields.</li>
+ * <li>Getter and setters for these fields.</li>
+ * <li>Static methods.</li>
  * </ul>
- * <p>
+ * <p/>
  * The following field data types are supported (no boxed types):
  * <ul>
- *   <li>boolean</li>
- *   <li>short</li>
- *   <li>int</li>
- *   <li>long</li>
- *   <li>float</li>
- *   <li>double</li>
- *   <li>byte[]</li>
- *   <li>String</li>
- *   <li>Date</li>
- *   <li>Any RealmObject subclass</li>
- *   <li>RealmList</li>
+ * <li>boolean</li>
+ * <li>short</li>
+ * <li>int</li>
+ * <li>long</li>
+ * <li>float</li>
+ * <li>double</li>
+ * <li>byte[]</li>
+ * <li>String</li>
+ * <li>Date</li>
+ * <li>Any RealmObject subclass</li>
+ * <li>RealmList</li>
  * </ul>
- * <p>
+ * <p/>
```

Comment:
These concern me. I guess this is why the JavaDoc flag was set in the Gradle file.
I agree this is correct HTML, but why does JavaDoc not accept it then. I am afraid that flag will also hide other more serious errors. If it stands between using `<p>` or disable the javadoc checks. I would much rather use `<p>`